src/server/app.ts

```diff
@@ -1,98 +1,99 @@
-    this.initializeMiddlewares();
-    this.initializeControllers();
-    this.initializeErrorHandling();
-    const PORT = config.get('www.port');
-    const server = this.expressApp.listen(PORT);
-    server.on('listening', EventListeners.onListening);
-    server.on('error', EventListeners.onError);
-    return server;
+    this.initializeMiddlewares()
+    this.initializeControllers()
+    this.initializeErrorHandling()
+    const PORT = config.get('www.port')
+    const server = this.expressApp.listen(PORT)
+    server.on('listening', EventListeners.onListening)
+    server.on('error', EventListeners.onError)
+    return server
   }
 
   public initializeMiddlewares() {
     i18n.configure({
       autoReload: true,
       directory: path.join(__dirname, '../locales'),
       objectNotation: true,
-    });
-    this.expressApp.use(i18n.init);
+    })
+    this.expressApp.use(i18n.init)
 
     this.expressApp.use((req, res, next) => {
-      res.setHeader('Access-Control-Allow-Origin', '*');
+      res.setHeader('Access-Control-Allow-Origin', '*')
       res.setHeader(
         'Access-Control-Allow-Methods',
-        'GET, POST, OPTIONS, PUT, PATCH, DELETE',
-      );
+        'GET, POST, OPTIONS, PUT, PATCH, DELETE'
+      )
       res.setHeader(
         'Access-Control-Allow-Headers',
-        'Content-Type, Authorization, Accept-Language',
-      );
-      next();
-    });
+        'Content-Type, Authorization, Accept-Language'
+      )
+      next()
+    })
 
     const LOGGER_CONSOLE = config.has('logger.console')
       ? config.get('logger.console')
-      : false;
+      : false
     if (LOGGER_CONSOLE) {
       this.expressApp.use(
         expressWinston.logger({
           winstonInstance: logger,
           meta: false,
-          msg:
-            "request - {{req.ip}} - {{res.statusCode}} - {{req.method}} - {{res.responseTime}}ms - {{req.url}} - {{req.headers['user-agent']}}",
+          msg: "request - {{req.ip}} - {{res.statusCode}} - {{req.method}} - {{res.responseTime}}ms - {{req.url}} - {{req.headers['user-agent']}}",
           expressFormat: false,
           colorize: true,
-        }),
-      );
+        })
+      )
     }
 
-    this.expressApp.use(bodyParser.json());
+    this.expressApp.use(bodyParser.json())
   }
 
   public initializeErrorHandling() {
-    this.expressApp.use(ErrorHandler.notFoundHandler);
-    this.expressApp.use(ErrorHandler.serverErrorHandler);
+    this.expressApp.use(ErrorHandler.notFoundHandler)
+    this.expressApp.use(ErrorHandler.serverErrorHandler)
     this.expressApp.use(
       expressWinston.errorLogger({
         winstonInstance: logger,
-      }),
-    );
+      })
+    )
   }
 
   public initializeControllers() {
     const controllers: BaseController[] = [
-      new AccountController(this.ctx),
-      new HealthCheckController(this.ctx),
-    ];
+      // new AccountController(this.ctx),
+      // new HealthCheckController(this.ctx),
+      new TodoItemController(this.ctx),
```

Comment:
Why comment health controller?

If you dont need account related functionality, please remove the code.